fastpay_core/tests/format.rs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #[test]
+#[ignore]
```

Comment:
I personally run `cargo fmt -- --config unstable_features=true --config imports_granularity=Crate` at every commit.

I noticed a bunch of clippy warnings: we should probably make all warnings fatal in CI.